rules-tests/Php82/Rector/Param/AddSensitiveParameterAttributeRector/Fixture/apply_attribute_to_functions.php.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rector\Tests\Php82\Rector\Param\AddSensitiveParameterAttributeRector\Fixture;
+
+function login($username, $password): void
+{
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php82\Rector\Param\AddSensitiveParameterAttributeRector\Fixture;
+
+function login($username, #[\SensitiveParameter]
```

Comment:
For compat with php7 the attribute should stay on its own line.

Reply:
new line can be added on param by reprint:

```php
$node->setAttribute(AttributeKey::ORIGINAL_NODE, null);
return $node
```

Reply:
Should/Could rector per default add attributes on its own line?

Reply:
Should/Could rector per default add attributes on its own line?

Reply:
No, because when auto add, existing attribute with no new line will automatically added, which incorrect if no change actually happen, that will cause a diff printing which should not.

Reply:
On site note: re-printing make other attribute gone on next rector call, so should be applied only when necessary, and make no traversable, see PR comment:

https://github.com/rectorphp/rector-src/pull/4191#issuecomment-1587337225

Reply:
Does a ECS rule exist to force attributes on separate lines?

Reply:
@staabm I would say this rule is useless for anyone using PHP 8.1 or less. I would be happy to change if you can suggest the best way to do it?

Reply:
@staabm I can't find any in ecs.
@peterfox for now, I think new line is not needed, as the `SensitiveParameter` attribute probably used only on App that not to be downgradeable. But if you want, you can add it by reprint it:

```php
$node->setAttribute(\Rector\NodeTypeResolver\Node\AttributeKey::ORIGINAL_NODE, null);
return $node
```

Reply:
@samsonasik thank you. I tried this code it didn't change the new line for me.

Reply:
let's just use single line for now then, we can improve in separate PR :)